src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const selectedTabId = tabs.some(tab => tab.id === activeTabId)
+    ? activeTabId
+    : tabs[0].id;
+
+  const activeTab = tabs.find(tab => tab.id === activeTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={tab.id === selectedTabId ? 'is-active' : ''}
+              data-cy="Tab"
+            >
+              <a
+                onClick={
+                  tab.id !== selectedTabId
+                    ? () => onTabSelected(tab.id)
+                    : undefined
+                }
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
```

Comment:
When mapping over the `tabs` array to render `<li>` elements, each element should have a `key` prop. This helps React optimize rendering and updates. Use a unique identifier like `tab.id` as the `key` prop for each `<li>`.